orca-webhook/src/main/groovy/com/netflix/spinnaker/orca/webhook/tasks/CreateWebhookTask.groovy

```diff
@@ -59,7 +59,14 @@ class CreateWebhookTask implements RetryableTask {
     def response
     try {
       response = webhookService.exchange(stageData.method, stageData.url, stageData.payload, stageData.customHeaders)
-    } catch (HttpStatusCodeException e) {
+    } catch(IllegalArgumentException e) {
```

Comment:
it's fascinating that it throws `IllegalArgumentException`....

Reply:
well it's really `validateURI` that wraps any exception in an `IllegalArgumentException`